template/styles/docfx.js

```diff
@@ -61,10 +64,29 @@ $(function () {
         } else {
           $(anchor).addClass('external-link');
         }
-    })
+      })
+  }
+
+  function addGtmButtons() {
+    $(".sample-container").each(function (i, block) {

```

Comment:
Since this method should add the paragraph to the first element with "sample-container" class I think it will be better if the jquery selector targets only the [first ](https://api.jquery.com/first/)one directly, without iterating through the whole collection of ".sample-container" elements.